prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudication-container/adjudication-container.component.ts

```diff
@@ -103,14 +103,21 @@ export class AdjudicationContainerComponent implements OnInit {
   }
 
   public onClaim(enrolleeId: number) {
+    const data: DialogOptions = {
+      data: { possibleAction: ClaimActionEnum.Claim },
+    };
+
+    this.busy = this.dialog.open(ClaimEnrolleeComponent, { data }).afterClosed().subscribe();
+
+    // TODO: Execute only if user performed Claim/Assign action
     this.adjudicationResource
       .setEnrolleeAdjudicator(enrolleeId)
       .subscribe((updatedEnrollee: HttpEnrollee) => this.updateEnrollee(updatedEnrollee));
   }
 
   public onDisclaim(enrolleeId: number) {
     const data: DialogOptions = {
-      title: 'Disclaim Enrolment',
+      data: { possibleAction: ClaimActionEnum.Disclaim },
```

Comment:
Feels like this should be `action` since this is the action that will be represented in the dialog